Detectors/Base/src/Aligner.cxx

```diff
@@ -24,7 +24,8 @@ using DetID = o2::detectors::DetID;
 
 DetID::mask_t Aligner::getDetectorsMask() const
 {
-  return DetID::getMask(mDetectors) & (~DetID::getMask(DetID::CTP));
+  return DetID::getMask(mDetectors) & (~DetID::getMask(DetID::CTP)) & (~DetID::getMask(DetID::FCT));
```

Comment:
Since you don't have alignable volumes at the moment, just a default (empty) placeholder object is enough, I've just uploaded it to http://alice-ccdb.cern.ch/browse/FCT/Calib/Align.
The defaults object is generated like in https://github.com/AliceO2Group/AliceO2/blob/dev/macro/UploadDummyAlignment.C (which would cover also FCT once this PR is merged, I've uploaded the object manually yo avoid re-generation of defaults for all detectors).
So, just remove this hack.

Reply:
Thanks a lot!

@Cas1997 I think we can then move ahead. Once you proposed all changes we can re-iterate your PR :)

Reply:
Done